spring-native/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  app:
+   #build: ./
+    image: otel-native-graalvm
+    environment:
+      OTEL_SERVICE_NAME: "graal-native-example-app"
+      OTEL_EXPORTER_OTLP_ENDPOINT: "http://collector:4317"
+      # Logs are disabled by default
+      OTEL_LOGS_EXPORTER: "otlp"
+    ports:
+      - "8080:8080"
+    depends_on:
+      - collector
+  collector:
+    image: otel/opentelemetry-collector-contrib:0.77.0
+    volumes:
+      - ./otel-config.yaml:/otel-config.yaml
+    command: ["--config=/otel-config.yaml"]
```

Comment:
My preference would be to call the collector's config something like `collector-config.yaml` or `otel-collector.yaml` so that users don't mistake this for an agent configuration.